core/src/main/java/com/linecorp/armeria/client/HttpClientPipelineConfigurator.java

```diff
@@ -441,6 +441,7 @@ public void onComplete() {}
             }, ctx.channel().eventLoop());
 
             // NB: No need to set the response timeout because we have session creation timeout.
```

Comment:
Let's move this comment one line down. 😄

Reply:
and done!